templates/es6/www

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+/**
+ * Entrypoint to application for es6 support
+ */
+ 
+require("babel-core/register");
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app').default;
```

Comment:
`babel-node` isn't supposed to be used in production either and using it here might make people do it - https://babeljs.io/docs/usage/cli/#babel-node.

How about moving the contents of `www` into a `server.js` and then `www` simply becomes:
```javascript
require('babel-core/register');
require('../server');
```
We can use ES2015 almost everywhere then. Or does that make the generating harder?